pkgs/development/libraries/physics/lhapdf/default.nix

```diff
@@ -6,7 +6,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "http://www.hepforge.org/archive/lhapdf/LHAPDF-${version}.tar.gz";
-    sha256 = "005jfapdj3mmk62p9qgvw7nyg93pqy249p1xy2ws1qx42xj76lih";
+    sha256 = "0gfjps7v93n0rrdndkhp22d93y892bf76pnzdhqbish0cigkkxph";
```

Comment:
@Mic92 
This happened: https://lhapdf.hepforge.org/hg/lhapdf/
I guess I jumped the gun here. Web pages still list mention 6.1.6 as the latest version: https://lhapdf.hepforge.org/